src/models/IDPDiscovery.js

```diff
@@ -66,14 +67,18 @@ function (Okta, PrimaryAuthModel, CookieUtil, Enums) {
               this.trigger('goToPrimaryAuth');
             }
             else {
+              var redirectFn = this.settings.get('features.redirectByFormSubmit')
```

Comment:
Wonder if we should move this logic inside the Util so we don't have to replicate everytime

Reply:
I figured it could be an derived property in `models/Settings.js`, sounds good?

Reply:
Yepp!